crfs.go

```diff
@@ -901,6 +903,7 @@ type node struct {
 	te *stargz.TOCEntry
 	sr *stargz.Reader
 	f  *os.File // non-nil if root & in debug mode
+	children map[string]fspkg.Node // Remenber child nodes once looked up.
```

Comment:
typo: remember

But more important is to document what the keys look like. It might make sense to use a before-line comment and spell it all out:

   // children maps from previously-looked up base names (like "foo.txt") to
   // the *node that was previously returned. This prevents FUSE inode numbers
   // from getting out of sync
   child map[string]*node

Also, I'd make the value type just be `*node`. It's slightly faster and smaller, but it's also more clear because that's the only type we'll have there, right?

Reply:
Thank you for your review. I fixed it on b076f18.